NICE.Identity.Management/ClientApp/src/views/User/User.tsx

```diff
@@ -1,20 +1,95 @@
-import React from "react";
+import React, { Component } from "react";
+
 import { RouteComponentProps, Link } from "react-router-dom";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { Panel } from "@nice-digital/nds-panel";
+import { Tag } from "@nice-digital/nds-tag";
+
+import { UserType } from "../../models/types";
+import { Endpoints } from "../../data/endpoints";
+import { UnlockUser } from "../../components/UnlockUser/UnlockUser";
 
 type TParams = { id: string };
 
 type UserProps = {} & RouteComponentProps<TParams>;
 
-export const User = (props: UserProps) => {
-	const { match } = props;
-	const { params } = match;
-
-	return (
-		<div className="grid">
-			<div data-g="12">
-				<p>User: {params.id}</p>
-				<Link to={`/users/${params.id}/roles`}>Roles</Link>
-			</div>
-		</div>
-	);
+type UserState = {
+	data: Array<UserType>;
+	error: string;
 };
+
+export class User extends Component<UserProps, UserState> {
+	constructor(props: UserProps) {
+		super(props);
+
+		this.state = {
+			data: [],
+			error: "",
+		};
+	}
+
+	updateData = (updatedData: Array<UserType>) => {
+		this.setState({ data: updatedData });
+	};
+
+	fetchData = async (url: string) => {
+		try {
+			const response = await fetch(url);
+			const data = await response.json();
+			this.setState({ data });
+		} catch (error) {
+			this.setState({ error });
+		}
+	};
+
+	componentDidMount() {
+		this.fetchData(Endpoints.user(this.props.match.params.id));
+	}
+
+	render() {
+		const { data, error } = this.state;
+		const userDetails = data[0];
+
+		return (
+			<>
+				<Breadcrumbs>
+					<Breadcrumb to="/" tag={Link}>
+						Home
+					</Breadcrumb>
+					<Breadcrumb to="/users" tag={Link}>
+						Users
+					</Breadcrumb>
+				</Breadcrumbs>
+
+				<div className="page-header" id="content-start">
+					<h1 className="page-header__heading">User details</h1>
+				</div>
+
+				{!error ? (
+					<div className="grid">
+						<div data-g="12 md:9" aria-busy={!data.length}>
+							{!data.length ? (
+								<p>Loading...</p>
+							) : (
+								<Panel>
+									<Tag>{!userDetails.blocked ? "Active" : "Locked"}</Tag>
+									<p>
+										User: {userDetails.first_name} {userDetails.last_name}
+									</p>
+
+									<UnlockUser
+										id={userDetails.id}
+										blocked={userDetails.blocked}
+										onToggleLock={this.updateData}
```

Comment:
I think we should be consistent across the codebase with how we create event listeners/callback props. Ie arrow function (like we currently have in UnlockUser.tsx or function reference like this. Preference but let's be consistent. I favour function references because I find it easier to follow but up to you.